src/components/Accordion/Accordion/__tests__/__snapshots__/accordion-snapshot-tests.jest.js.snap

```diff
@@ -10,22 +10,23 @@ exports[`Accordion renders correctly with allowMultiple 1`] = `
     id="monday-accordion--item-0"
   >
     <div
-      className="expand-collapse--wrapper"
+      className="expandCollapseWrapper expand-collapse--wrapper"
+      data-testid="expand-collapse_expand-collapse--monday-accordion--item-0"
```

Comment:
convention is right, that's how data-testid is specified in ExpandCollapse
`data-testid={dataTestId || getTestId(ELEMENT_TYPES.EXPAND_COLLAPSE, id)}`
`id` which `AccordionItem` is adding is too verbose and need to be fixed, but I'd rather skip it for now - it might require tests changes and that's out of the scope of this task